brands/shop/convenience.json

```diff
@@ -780,8 +780,15 @@
   },
   "shop/convenience|Martin's": {
     "count": 53,
+    "countryCodes": ["gb"],
+    "match": [
+      "shop/newsagent|Martin's",
+      "shop/newsagent|McColls"
+    ],
```

Comment:
Well, if the entry is the same as `shop/newsagent|Martin's` and `shop/newsagent|McColls` then why did you created another entry for that in the `brands/shop/newsagent.json` file?

Reply:
According to https://en.wikipedia.org/wiki/McColl%27s the Martin's store should be tagged as `shop/newsagent|Martin's` and not as `shop/convenience|Martin's`. So I would tend to remove this entry and leave the other one (with the correct match)